radis/lbl/loader.py

```diff
@@ -1323,21 +1336,49 @@ def fetch_databank(
                 )
             if len(frames) > 1:
                 # Note @dev : may be faster/less memory hungry to keep lines separated for each isotope. TODO : test both versions
-                for df in frames:
-                    if "iso" not in df.columns:
-                        assert "iso" in df.attrs
-                        df["iso"] = df.attrs["iso"]
-                # Keep attributes:
-                from radis.misc.basics import intersect
-
-                attrs = frames[0].attrs
-                for df in frames[1:]:
-                    attrs = intersect(attrs, df.attrs)
-                del attrs["iso"]  # added as a column (different for each line)
-                # Merge:
-                df = pd.concat(frames, ignore_index=True)  # reindex
-                df.attrs = attrs
-                self.params.dbpath = ",".join(local_paths)
+                if output == "pandas":
+                    for df in frames:
+                        if "iso" not in df.columns:
```

Comment:
```suggestion
                        # to save some memory, we have added "iso" as an attribute rather than a column if it is the same for all lines 
                        if "iso" not in df.columns:
```